.cascade-code/Chapman.edu/_components/omni_nav_v2/omni_nav.vtl

```diff
@@ -123,10 +123,25 @@
   #set ( $umbrellaFolder = $_.locateFolder($umbrellaDirectoryPath) )
 
   #foreach( $childAsset in $umbrellaFolder.children )
-    #set ( $metaHide = $childAsset.metadata.getDynamicField('Hide from navigation').value )
-
-    ## Skip if hidden or is index page.
-    #if ( $metaHide != 'Yes' && $childAsset.name != 'index' )
+    ## When the 'Hide from navigation' box is not checked, and it has never been checked, the metadata 
+    ## field doesn't exist for that asset. When the box is not checked, but it was at one point, it exists with a value of ''.
+    ## See Github issue: https://github.com/chapmanu/cascade-assets/issues/262
+    ## By default, set visibility to false
+    #set ( $isVisibleChild = false )
```

Comment:
This is essentially the same code used to get unhidden assets in the left nav, which didn't hit the issues that the original code did. See issue #262